aspnet/mvc/views/partial.rst

```diff
@@ -1,8 +1,128 @@
-.. include:: /../common/stub-topic.txt
+Partial Views
+=============
 
-|stub-icon| Partial Views
-=========================
+By `Steve Smith`_
+
+ASP.NET Core MVC supports partial views, which are useful when you have reusable parts of web pages you want to share between different views.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+`View or download sample code <https://github.com/aspnet/Docs/tree/master/aspnet/mvc/views/partial/sample>`__
```

Comment:
>View or download sample code 
@danroth27  I've been moving this to the additional resources section at the bottom. But it might be better to leave it here.